packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -167,6 +191,8 @@ export class DetailsColumnBase extends BaseComponent<IDetailsColumnProps> {
       this._dragDropSubscription.dispose();
       delete this._dragDropSubscription;
     }
+    this._async.dispose();
+    this._events.dispose();
```

Comment:
Out of scope for this PR, but it'd be really cool to pass a finalizer to these utilities when they are initialized like React does with Hooks.